jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/controller/AdministrationController.java

```diff
@@ -332,6 +322,16 @@ public ModelAndView getSpamProtectionPage(){
         return new ModelAndView("spamProtection").addObject("rules", ruleDtos);
     }
 
+    @RequestMapping(value = "/usersgroup/list", method = RequestMethod.GET)
```

Comment:
Done. As we already have method mapped to `/user` endpoint I used mapping by parameters.